bitsv/network/services/bchsvexplorer.py

```diff
@@ -45,15 +46,15 @@ def get_address_info(cls, address):
 
     @classmethod
     def get_balance(cls, address):
-        r = requests.get(cls.MAIN_BALANCE_API.format(address), timeout=DEFAULT_TIMEOUT)
+        r = requests.get(cls.MAIN_ADDRESS_BALANCE.format(address), timeout=DEFAULT_TIMEOUT)
         r.raise_for_status()  # pragma: no cover
-        return r.json()
+        return int(r.json()['balance'])
 
     @classmethod
     def get_transactions(cls, address):
-        r = requests.get(cls.MAIN_ADDRESS_API.format(address), timeout=DEFAULT_TIMEOUT)
+        r = requests.get(cls.MAIN_ADDRESS_TX_IDS.format(address), timeout=DEFAULT_TIMEOUT)
         r.raise_for_status()  # pragma: no cover
-        return r.json()['transactions']
+        return r.json()['txids']
```

Comment:
I will change this to:

```
        txs = r.json().get('txids')  # if there are no txs in history the 'txids' key doesn't exist
        if txs:
            return txs
        return []
```

Because wierdly their api drops the 'txids' key if there is no history for they key/address...

Reply:
then `return r.json().get('txids', [])` would suffice